src/client.ts

```diff
@@ -3,11 +3,29 @@ import { metricsDefinitions } from './metrics'
 import { SnapshotsFetcherComponents } from './types'
 import { contentServerMetricLabels, fetchJson, saveContentFileToDisk as saveContentFile } from './utils'
 
-export async function getGlobalSnapshot(components: SnapshotsFetcherComponents, server: string, retries: number) {
-  const url = new URL(`${server}/snapshot`).toString()
+export async function getGlobalSnapshots(components: SnapshotsFetcherComponents, server: string, retries: number):
+  Promise<{
+    hash: string,
+    lastIncludedDeploymentTimestamp: number
+  }[]> {
+  try {
+    const newUrl = new URL(`${server}/snapshots`).toString()
+    // TODO: validate response
+    const newSnapshots: {
+      hash: string,
+      timeRange: { initTimestampSecs: number, endTimestampSecs: number }
+    }[] = await components.downloadQueue.scheduleJobWithRetries(() => fetchJson(newUrl, components.fetcher), 1)
+    return newSnapshots.map(newSnapshot => ({
```

Comment:
what's the order of newSnapshots array? I'd like to process first the most recent snapshot batch, right?

Reply:
Added